docs/example_usage/breeds_datasets.rst

```diff
@@ -0,0 +1,288 @@
+(e.g., animal breeds) present in the training set (source domain) are 
+entirely different from the test set (target domain). 
+You can create BREEDS datasets from any existing dataset that has 
+an explicit class hierarchy describing the semantic relationship 
+between pairs of classes (e.g., ImageNet, OpenImages). 
+In this walkthrough, we will do so using ImageNet and the corresponding
+class hierarchy from [STM20]_ (modification of WordNet to be better-suited
+for the object recognition task).
+
+At a high-level, the procedure to construct BREEDS datasets is as follows:
+
+1. Use the dataset's class hierarchy to define `superclasses` by grouping
+   together semantically similar dataset classes (e.g., define the `dog`` superclass 
+   by aggregating all the dog breed classes). 
+2. Design the classification task to be between different superclasses, so that
+   we have easy access to the individual subpopulations (i.e., the original 
+   dataset classes). Then, for each superclass, we will partition the subpopulations 
+   into the source (used for model training) and target domain without overlap
+   (used for evaluation). 
+
+
+.. raw:: html
+
+   <i class="fa fa-play"></i> &nbsp;&nbsp; <a
+   href="https://github.com/MadryLab/BREEDS-Benchmarks/blob/master/Constructing%20BREEDS%20datasets.ipynb">Download
+   a Jupyter notebook</a> containing all the code from this walkthrough! <br />
+   <br />
+
+Requirements/Setup
+''''''''''''''''''
+To create BREEDS datasets using ImageNet, we need to create a: 
+
+- ``data_dir`` which contains the dataset (in this case, ImageNet) 
+  in PyTorch-readable format.
+- ``info_dir`` which contains information about the class hierarchy.
+  For ImageNet, you can find the relevant files in the 
+  `imagenet_class_hierarchy/modified` subfolder of our
+  `release <https://github.com/MadryLab/BREEDS_benchmarks>`_. 
+
+Part 1: Browsing through the Class Hierarchy
+''''''''''''''''''''''''''''''''''''''''''''
+The :class:`~robustness.tools.breeds_helpers.ClassHierarchy` class allows
+us to probe the class hierarchy. The class hierarchy has a graph structure,
+where a child node is a subclass (an instance) of the parent node and the
+original dataset classes are the leaves. We use a top-down approach to obtain 
+superclasses. Specifically, by defining all nodes as a fixed (user-specified) 
+distance from the root node as superclasses, and treating all the descendant 
+leaves (dataset classes) as subpopulations. 
+
+To browse through the ImageNet class hierarchy, let's create an instance 
+of the ``ClassHierarchy`` class:
+
+.. code-block:: python
+
+   from robustness.tools.breeds_helpers import ClassHierarchy
+   hier = ClassHierarchy(info_dir)
+
+
+Here, :samp:`info_dir` should be the path to a folder which contains
+information about the class hierarchy 
+(e.g., ``imagenet_class_hierarchy/modified``). In general, :samp:`info_dir`
+must contain the following files:
```

Comment:
Just say that info_dir is the repo you downloaded in the previous step, this part is confusing